### DISCUSSION TITLE
Wrong link on dashboard

### DISCUSSION THREAD

**Discussion by zerosuka** [teslamate-org/teslamate#2466] [Q&A] [answered]
Hi guys,
does someone know hot to fix this wrong link if mouse point to teslamate icon?
I moved from a lan (192.168.**47**.XXX) to another one (192.168.**1**.XXX).
All works correctly, 192.168.1.30:3000 works, 192.168.1.30:4000 works.
Clicking on teslamate icon, it redirect to the wrong address (192.168.**47**.XXX:4000)

<a href="https://ibb.co/NFmyW1X"><img src="https://i.ibb.co/K5LrVNY/Immagine.png" alt="Immagine" border="0"></a>

Thank you a lot

**Comment by bahree** [comment, marked as answer]
In TeslaMate ==> Settings (not Grafana) - scroll all the way to the bottom and you have two URLs - one for webapp and one for dashboards.

![image](https://user-images.githubusercontent.com/3529468/153731740-ae76a1e0-57dc-4087-b898-d4f89b32218e.png)

**Comment by zerosuka** [comment]
**I LOVE U MAN!!!!**
